admiral/pkg/clusters/shard_handler.go

```diff
@@ -35,11 +35,12 @@ func (sh *ShardHandler) Added(ctx context.Context, obj *admiralapiv1.Shard) erro
 }
 
 func (sh *ShardHandler) Deleted(ctx context.Context, obj *admiralapiv1.Shard) error {
-	// TODO: Not yet implemented
 	//err := HandleEventForShard(ctx, admiral.Delete, obj, sh.RemoteRegistry)
```

Comment:
was this intentional ?

Reply:
Yes, there is nothing to do in handler for delete - the controller removes from cache, and we don't want operator to try to recreate the shard or anything like that